aten/src/ATen/native/Fill.cpp

```diff
@@ -40,7 +40,7 @@ Tensor& fill_out(Tensor& self, const Scalar& value) {
 static Tensor& fill_out_quantized(Tensor& self, const Scalar& value) {
   at::Tensor out = at::ones(self.sizes()).to(kFloat) * value;
```

Comment:
btw this looks like allocating quite a lot :) given that `self` will get filled, and `out` thrown away

Reply:
agreed but I don't really know where out_quantized is being sued from and the reasons why it was written in this way

Reply:
that being said: https://github.com/pytorch/pytorch/pull/109544